deps/wazuh_testing/wazuh_testing/modules/aws/data_generator.py

```diff
@@ -0,0 +1,1185 @@
+                    'eventSource': 'sts.amazonaws.com',
+                    'eventName': 'AssumeRole',
+                    'awsRegion': cons.US_EAST_1_REGION,
+                    'sourceIPAddress': 'ec2.amazonaws.com',
+                    'userAgent': 'ec2.amazonaws.com',
+                    'requestParameters': {
+                        'roleArn': f"arn:aws:iam::{cons.RANDOM_ACCOUNT_ID}:role/demo-415-v2-InstanceRole-1FB0FMP2EXOKN",
+                        'roleSessionName': 'i-0e9ddef5daf05c7df'
+                    },
+                    'responseElements': {
+                        'credentials': {
+                            'accessKeyId': 'ASIASNL6BLJL7ZA3J6WP',
+                            'sessionToken': str(uuid4()),
+                            'expiration': 'Dec 23, 2021, 3:51:35 PM'
+                        }
+                    },
+                    'requestID': str(uuid4()),
+                    'eventID': str(uuid4()),
+                    'readOnly': True,
+                    'resources': [
+                        {
+                            'accountId': cons.RANDOM_ACCOUNT_ID,
+                            'type': 'AWS::IAM::Role',
+                            'ARN': f"arn:aws:iam::{cons.RANDOM_ACCOUNT_ID}:role/demo-415-v2-InstanceRole-1FB0FMP2EXOKN"
+                        }
+                    ],
+                    'eventType': 'AwsApiCall',
+                    'managementEvent': True,
+                    'eventCategory': 'Management',
+                    'recipientAccountId': cons.RANDOM_ACCOUNT_ID,
+                    'sharedEventID': str(uuid4())
+                }
+            ]
+        })
+
+
+class VPCDataGenerator(DataGenerator):
+    BASE_PATH = join(cons.AWS_LOGS, cons.RANDOM_ACCOUNT_ID, cons.VPC_FLOW_LOGS, cons.US_EAST_1_REGION)
+    BASE_FILE_NAME = f'{cons.RANDOM_ACCOUNT_ID}_{cons.VPC_FLOW_LOGS}_{cons.US_EAST_1_REGION}_'
+
+    def get_filename(self) -> str:
+        """Return the filename in the VPC format.
+
+        Example:
+            <prefix>/AWSLogs/<suffix>/<organization_id>/<account_id>/vpcflowlogs/<region>/<year>/<month>/<day>
+
+        Returns:
+            str: Synthetic filename.
+        """
+        now = datetime.utcnow()
+        path = join(self.BASE_PATH, now.strftime(cons.PATH_DATE_FORMAT))
+        name = (
+            f'{self.BASE_FILE_NAME}{cons.FLOW_LOG_ID}_{now.strftime(cons.FILENAME_DATE_FORMAT)}_{abs(hash(now))}'
+            f'{cons.LOG_EXT}'
+        )
+
+        return join(path, name)
+
+    def get_data_sample(self) -> str:
+        """Return a sample of data according to the VPC format.
+
+        Returns:
+            str: Synthetic data.
+        """
+        data = [
+            [
+                "version", "account-id", "interface-id", "srcaddr", "dstaddr", "srcport", "dstport", "protocol",
+                "packets", "bytes", "start", "end", "action", "log-status"
+            ]
+        ]
+
+        for _ in range(5):
```

Comment:
It's a convention in Python to use `_` for throwaway variables. It just indicates that the loop variable isn't actually used.
I choose this way because I think is more readable than a comprehension list. Do you agree with that?

Reply:
Also, this applies to all the occurrences in the file.